### PR TITLE
feat: Add helm chart files (with some values hardcoded yet)

### PR DESCRIPTION
**Pull request by ruimartinsptl**
Add the initial helm chart files.

This first pull request contains the minimum files generated using the `helm create` and some values hardcoded, like the image version, my idea is to iterate this pull request creating new pull requests improving the helm file, and improving the README file...

If the reviewers prefer that this MR will be already with variables, etc, I need more time to conclude this task

### CODE REVIEW

File: Chart.yaml
Comment (fliiiix) [inline Chart.yaml:3]: should that contain a description what this heml chart does?

**Review by fliiiix — APPROVED** [review]
i have no idea how helm charts work but lgtm :tada:

File: values.yaml
Comment (timofurrer) [inline values.yaml:12]: I think you want to remove the default `tag` here, so that app version is used in the stateful set.

File: Chart.yaml
Comment (ruimartinsptl) [reply Chart.yaml:3]: I updated the Pull Request description :) This was generated automatically, it's to be replaced, but I can update on this pull request if you want :)

File: values.yaml
Comment (ruimartinsptl) [reply values.yaml:12]: Yap, I should :) Should I do that on this merge request, or should I do that in the next iteration?

This first MR is working, is what I'm using on my company, but is the minimum to have this working, I pretend to submit some improvements in the next weeks, but for today I only have this hardcoded for now :)

Comment (timofurrer) [reply values.yaml:12]: If the change happens asap, that's fine with me. 

@fliiiix up to you to merge.

Comment (fliiiix) [reply values.yaml:12]: @ruimartinsptl up to you whatever makes more sense to you, you should be able to merge yourself :)

Comment (fliiiix) [reply values.yaml:12]: image


yep so feel free to do everything a maintainer would do :)

Comment (ruimartinsptl) [reply values.yaml:12]: Ok, thanks :)

**Pull request merged**
